src/software/thunderscope/constants.py

```diff
@@ -271,6 +275,10 @@ class Colors(object):
     NAVIGATOR_OBSTACLE_COLOR = QtGui.QColor(255, 80, 0, 100)
     DEBUG_SHAPES_COLOR = QtGui.QColor(190, 50, 235, 255)
     PASS_VISUALIZATION_COLOR = QtGui.QColor(255, 0, 0, 80)
+    UNCOMMITTED_PASS_VISUALIZATION_COLOR = QtGui.QColor(255, 0, 0, 80)
```

Comment:
we should start using a `ColourMap` at some point for better colour theming: https://pyqtgraph.readthedocs.io/en/latest/api_reference/colormap.html

Reply:
Im not very familiar with `ColourMap`, @williamckha would likely know more since I think he used it for the pass layer, but from quickly reading about it it sounds like it's mainly used for mapping a value to a color (e.g. the pass score to a color). In this case, these are static colors that Im using. Unless you're suggesting that we should change the color of the pass line based on its score.

Reply:
I haven't thought about imementation details but that's definitely one use case that you highlighted.

I was thinking about another use case where the colour pallette of thunderscope becomes a bit more unified using ColorMaps. Some ColorMaps target data visualization (0-1 mapping) but others give a nice pallette that we could consider using: https://matplotlib.org/stable/users/explain/colors/colormaps.html#qualitative

It might be something to think about in the future

Reply:
Maybe someone can take it on as next year as a first ticket. But if we're serious about rewriting thunderscope then this might be too trivial to worry about